src/main/res/values/strings.xml

```diff
@@ -734,7 +734,7 @@
     -->
     <string name="location_china_mainland">Mainland China</string>
     <string name="location_christchurch_nz">Christchurch, New Zealand</string>
-    <string name="location_crimea">Crimea</string>
+    <string name="location_crimea">Crimea, Eastern Europe</string>
```

Comment:
Just leave it unspecified

Reply:
That looks extra problematic. The main problem is with other places being also called "Crimea", (which is now solved)
https://en.wikipedia.org/wiki/Crimea_Pass
https://en.wikipedia.org/wiki/Darvils,_Virginia
https://en.wikipedia.org/wiki/Crimea,_Louisiana

Reply:
Other places are minor in comparison (Both US villages are small and unincorporated) or named sufficiently differently (Crimea Pass).
The minor risk of user confusion is very small compared with potential to offend people needlessly or get into political dispute.
Confusion for translators is also unlikely but feel free to add link to https://en.wikipedia.org/wiki/Crimea which btw is just "Crimea"

Reply:
It does say
> Geography
> --
> Eastern Europe

It did confuse me. Now just "Crimea", and I can comment the actual string instead.